diff -Nur -x '*.orig' -x '*.rej' tt-rss-1.6.1/include/sanity_check.php mezzanine_patched_tt-rss-1.6.1/include/sanity_check.php
--- tt-rss-1.6.1/include/sanity_check.php	2012-11-14 21:45:12.420295677 +0100
+++ mezzanine_patched_tt-rss-1.6.1/include/sanity_check.php	2012-11-14 21:44:58.202033947 +0100
@@ -20,10 +20,6 @@
 				array_push($errors, "PHP version 5.3.0 or newer required.");
 			}
 
-			if (CONFIG_VERSION != EXPECTED_CONFIG_VERSION) {
-				array_push($errors, "Configuration file (config.php) has incorrect version. Update it with new options from config.php-dist and set CONFIG_VERSION to the correct value.");
-			}
-
 			if (!is_writable(CACHE_DIR . "/images")) {
 				array_push($errors, "Image cache is not writable (chmod -R 777 ".CACHE_DIR."/images)");
 			}
