diff -Nur Tiny-Tiny-RSS-20161005.git6687cb9.old/include/sanity_check.php Tiny-Tiny-RSS-20161005.git6687cb9.new/include/sanity_check.php
--- Tiny-Tiny-RSS-20161005.git6687cb9.old/include/sanity_check.php	2016-10-05 23:25:33.000000000 +0200
+++ Tiny-Tiny-RSS-20161005.git6687cb9.new/include/sanity_check.php	2016-10-05 23:45:04.626231133 +0200
@@ -47,10 +47,6 @@
 				array_push($errors, "PHP version 5.4.0 or newer required.");
 			}
 
-			if (CONFIG_VERSION != EXPECTED_CONFIG_VERSION) {
-				array_push($errors, "Configuration file (config.php) has incorrect version. Update it with new options from config.php-dist and set CONFIG_VERSION to the correct value.");
-			}
-
 			if (!is_writable(CACHE_DIR . "/images")) {
 				array_push($errors, "Image cache is not writable (chmod -R 777 ".CACHE_DIR."/images)");
 			}
